opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
Just pushed an update, that made your comments hidden.  I basically changed the name to just `rpc.service`, with the intention of making these names symmetrical, independent of client / server side, They just refer to the 'peer' of the current span's role, I think it makes it cleaner and doesn't pollute the tags namespace, what do you think?

I also don't like Zipkin's way of merging client and server spans into one. On the plus side, current API works even with Zipkin, as I have just proven to myself by migrating our Python client to a Python version of opentracing API (fyi - I am waiting for Uber OSS team approval to make the repo public). Changing Zipkin to two-span model will be somewhat painful, I think there's a lot of assumptions, especially in the UI, based on that model. But I think @adriancole might be up to it with v2 initiative :-)

But I think we may need some additional tag, like `is_client`, to make the tracer distinguish client-side from server-side spans. Without it Zipking tracer can't emit the right ss/sr/cs/cr annotations. In most cases Zipkin tracer can infer it by assuming start/join-trace are server-side, and start_child is client-side, but there are cases when start-trace needs to be client-side. My previous API was allowing an `is_client` flag in the start_trace method.